### PR TITLE
log analytics events based on pixi searches

### PR DESCRIPTION
**Pull request by patrickkettner**
meant to collect information as narrowly as possible, let me know if it should change

### CODE REVIEW

File: pixi/backend/api.js [hunk 1]
```diff
@@ -88,6 +90,34 @@ const execChecks = async (url) => {
...
+      // Application Name - for easier searching in GA
+      an: 'pixi',
+      // Event Action - required value
+      ea: 'lint',
+      // Event Label - where the URL is stored
+      el: new URL(url).host,
+    };
+
+    return fetch(
+      `http://www.google-analytics.com/collect?${new URLSearchParams(data)}`
```
Comment (sebastianbenz) [inline pixi/backend/api.js:114]: `http`?

File: pixi/backend/api.js [hunk 2]
```diff
@@ -97,6 +127,7 @@ api.get('/lint', async (request, response) => {
 
   const fetchUrl = request.query.url;
   try {
+    await logAnalytics(fetchUrl);
```
Comment (sebastianbenz) [inline pixi/backend/api.js:130]: Why block on logging?

File: pixi/backend/api.js (hunk 1 above)
Comment (patrickkettner) [reply pixi/backend/api.js:114]: whoopsie!

File: pixi/backend/api.js (hunk 2 above)
Comment (patrickkettner) [reply pixi/backend/api.js]: thanks! vestigial code, removed the await in the last update

**Comment by nainar** [comment]
OUR Privacy disclosure covers this given we are not collecting any data about the user other than the timestamp and the domain/URL entered. LGTM from me

**Review by nainar — APPROVED** [review]
See comment on thread

**Review by sebastianbenz — APPROVED** [review]

**Pull request merged**